.github/workflows/builder.yml

```diff
@@ -117,12 +123,6 @@ jobs:
         run: |
           echo "${{ github.sha }};${{ github.ref }};${{ github.event_name }};${{ github.actor }}" > rootfs/OFFICIAL_IMAGE
 
-      - name: Login to DockerHub
-        uses: docker/login-action@v1.12.0
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-
       - name: Login to GitHub Container Registry
```

Comment:
I don't think it makes sense to move the login of the DockerHub registry, but leave the GitHub Registry behind. I suggest moving those in pairs.